doc/build/contribute_to_paddle.md

```diff
@@ -20,16 +20,30 @@ It's just that simple.
 
 ## Clone
```

Comment:
In original doc, 'Your code mush be fully documented by doxygen style.',  mush -> must?